src/views/auth/Signin.vue

```diff
@@ -23,18 +22,27 @@ import { ActionBindingIF } from '@/interfaces'
 export default class SigninView extends Vue {
   @Action setAuthenticated: ActionBindingIF
 
-  // Called when keycloak-session-ready event is emitted which indicates the user is authenticated
-  private async redirectOnLogin () {
+  private tokenService = new TokenService()
```

Comment:
I really need to test whether the refresh timer keeps working when this component is destroyed.